backend/core/src/main/java/org/sonarsource/sonarlint/core/telemetry/TelemetryServerAttributesProvider.java

```diff
@@ -78,10 +87,27 @@ public TelemetryServerAttributes getTelemetryServerLiveAttributes() {
 
     var nodeJsVersion = getNodeJsVersion();
 
-    return new TelemetryServerAttributes(usesConnectedMode, usesSonarCloud, devNotificationsDisabled, nonDefaultEnabledRules,
+    return new TelemetryServerAttributes(usesConnectedMode, usesSonarCloud, sonarQubeServerBindingCount,
+      sonarQubeCloudEUBindingCount, sonarQubeCloudUSBindingCount, devNotificationsDisabled, nonDefaultEnabledRules,
       defaultDisabledRules, nodeJsVersion);
   }
 
+  private int countSonarQubeCloudBindings(Collection<BoundScope> allBindings, SonarCloudRegion region) {
+    return allBindings.stream()
+      .mapToInt(binding -> {
+        if (connectionConfigurationRepository.getConnectionById(binding.getConnectionId()) instanceof SonarCloudConnectionConfiguration scBinding) {
+          return region.equals(scBinding.getRegion()) ? 1 : 0;
+        }
+        return 0;
```

Comment:
IMHO this could also be implemented as a `filter` + `count` for consistency and readability, or did I miss anything?

Reply:
Yes, I had it like that initially but it did not look very good :D I will put it back